java/sigmf-ns-ntia.schema

```diff
@@ -142,11 +164,14 @@
         "mobile" : {
           "type" : "boolean"
         },
-        "location" : {
-          "$ref" : "#/definitions/Location"
+        "geolocation" : {
```

Comment:
Done in d6c7e4893d71a82f72bd01f99b455ed4146aa0c1 (which also fixes the global field name, which should be "core:geolocation" instead of simply "geolocation")

Reply:
I've deprecated it in the code so it shouldn't be removed.

Reply:
Understood- reverted my change in f1da7dd967a6ca68e736f65794eb00d2450c5d8d